docs/content/users/install/ddev-installation.md

```diff
@@ -180,115 +174,62 @@ Once you’ve [installed a Docker provider](docker-installation.md), you’re re
-
-        ```powershell
-        Set-ExecutionPolicy Bypass -Scope Process -Force; [System.Net.ServicePointManager]::SecurityProtocol = [System.Net.ServicePointManager]::SecurityProtocol -bor 3072;
-        iex ((New-Object System.Net.WebClient).DownloadString('https://raw.githubusercontent.com/ddev/ddev/main/scripts/install_ddev_wsl2_docker_inside.ps1'))
-        ```
-    3. In *Windows Update Settings* → *Advanced Options* enable *Receive updates for other Microsoft products*. You may want to occasionally run `wsl.exe --update` as well.
-
-    Now you can use the "Ubuntu" terminal app or Windows Terminal to access your Ubuntu distro, which has DDEV and Docker working inside it.
-
-    ### WSL2 + Docker Desktop Install Script
+    * Installing WSL2 and installing an Ubuntu distro (or some other distro if you're installing manually).
+    * Optionally installing Docker Desktop or Rancher Desktop for Windows and enabling WSL2 integration with the distro (if you're using the Docker/Rancher Desktop approach).
+    * Running the DDEV installer and choosing the approach you want.
 
-    WSL2 with Docker Desktop is a less-favored choice because Docker Desktop may be lightly supported, and has many features not required for use with DDEV that do not add particular value. It is also not free software (although smaller organizations can use it free of charge) and it is not open-source.
+    ### Windows Installer
 
-    The script here prepares your default WSL2 Ubuntu distro for use with Docker Desktop, and you can run the script multiple times without breaking anything.
+    The easiest way to install DDEV on Windows is to use the DDEV Windows installer, which can handle different installation scenarios:
 
-    In all cases:
+    1. **Download the Windows installer** from the [DDEV releases page](https://github.com/ddev/ddev/releases).
+    2. **Run the installer** and choose your installation type:
 
-    1. Install WSL2 with an Ubuntu distro. On a system without WSL2, run:
-        ```powershell
-        wsl --install
-        ```
+           - **Docker CE inside WSL2** (Recommended): The installer will automatically install Docker CE in your WSL2 environment. This is the fastest and most reliable option.
+           - **Docker Desktop/Rancher Desktop**: Choose this if you already have Docker Desktop or Rancher Desktop installed or prefer to use them.
+           - **Traditional Windows**: Choose this for non-WSL2 installations (requires Docker Desktop or Rancher Desktop). Note that both Docker Desktop and Rancher Desktop themselves use WSL2, but you will interact with DDEV in Git Bash, PowerShell, or Cmd on the Windows side.
```

Comment:
> Note that both Docker Desktop and Rancher Desktop themselves use WSL2

I installed Docker Desktop in a virtual machine using WSL2 and Hyper-V methods, and DDEV worked in both cases. Do we recommend using WSL2 backend all the time?

Reply:
WSL2 is the recommendation of Docker, and Hyper-V is the old way. I'm fine with people using both, thanks for testing both. But "choice is a sickness of the mind"